gpMgmt/test/behave/mgmt_utils/gpperfmon.feature

```diff
@@ -90,6 +90,15 @@ Feature: gpperfmon
         Then wait until the results from boolean sql "SELECT count(*) = 0 FROM queries_history WHERE query_text like '--alter distributed by%'" is "true"
         And wait until the results from boolean sql "SELECT count(*) = 1 FROM queries_history WHERE query_text like '--end flag%'" is "true"
 
+    @gpperfmon_query_history
+    Scenario: gpperfmon adds time-consuming queries with different executed statement to queries_now and queries_history
+        Given gpperfmon is configured and running in qamode
+        When the user truncates "queries_history" tables in "gpperfmon"
+        Then run function with query from table for 80 seconds
+        And wait until the results from boolean sql "SELECT count(*) = 1 FROM queries_now WHERE query_text like '%where pg_sleep(80) is not null'" is "true"
```

Comment:
do we need to check queries_now? the query text gets into queries_now without a patch.

Reply:
> the query text gets into queries_now without a patch
Can't confirm

While developing, I faced the situation when query text reached the queries_history but was missing from queries_now